GLOSSARY.md

```diff
@@ -0,0 +1,22 @@
+# Cog Glossary
+
+## Bundle
+## Command
+### Subcommand
+## Group
+## Permission
+  * A way to define what commands can be accessed
+  * Come with bundles
+  * Can be created custom in the `site:` namespace
+
+### Examples:
+  * `operable:manage_commands`
+  * `ec2:delete`
+
+## Relay
+## Role
+  * Group of permissions
```

Comment:
Got it. I'd prefer to keep the glossary specifically focused on definitions (ie: "named collection of permissions") and avoid going into our intent.